docker-compose/docker-compose.multiserver.mariadb.yml

```diff
@@ -62,8 +73,15 @@ services:
       - ACTIVEMQ_BROKER_PASSWORD=password
     ports:
       - "8080:8080"
+    depends_on:
+      - mariadb
+      - activemq
     volumes:
-      - data:/data
+      - ./data:/data
```

Comment:
Why the change to a local mount?

Reply:
It gives you the advance to look into the events files, but for this case is not needed.

I rolled back to the original statement.